src/main/java/com/sphy/hotelmanagementapplication/domain/User.java

```diff
@@ -0,0 +1,139 @@
+package com.sphy.hotelmanagementapplication.domain;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+@Entity
+@Inheritance(strategy = InheritanceType.JOINED)
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private boolean EmailVerify;
+    private String username;
+    private String firstname;
+    private String lastname;
+    private String email;
+    private String hashedPassword;
```

Comment:
You should add the property password of type String with the annotation @Transient, so that it won't be stored to the database